cpp/benchmarks/io/parquet/parquet_writer.cpp

```diff
@@ -202,8 +202,8 @@ NVBENCH_BENCH_TYPES(BM_parq_write_encode, NVBENCH_TYPE_AXES(d_type_list))
   .set_name("parquet_write_encode")
   .set_type_axes_names({"data_type"})
   .set_min_samples(4)
-  .add_int64_axis("cardinality", {0, 1000})
-  .add_int64_axis("run_length", {1, 32});
+  .add_int64_axis("cardinality", {1, 1000, 10'000, 100'000, 1'000'000})
```

Comment:
Are there any resource/infrastructure constraints that we need to consider here? We can perhaps remove one or two entries from here if needed.

Reply:
I think we can drop `1`. Given the table size, it's effectively the same as 1000.
Also, should 1M just be 0? Not sure if the goal is to have unique elements (AFAIK row count is typically lower than 1M).